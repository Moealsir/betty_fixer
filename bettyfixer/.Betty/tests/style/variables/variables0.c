int i;
int global_var;
	int indented_global = 98;
unsigned int another = 12;
void (*global_func_ptr)(int, int);

int test_func(int some_var,
	int (*func)(int),
	test_t *some_other)
{
	return (0);
}
