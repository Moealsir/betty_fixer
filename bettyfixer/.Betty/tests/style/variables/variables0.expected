tests/style/variables/variables0.c:1: ERROR: global variables are not allowed
tests/style/variables/variables0.c:2: ERROR: global variables are not allowed
tests/style/variables/variables0.c:3: ERROR: global variables are not allowed
tests/style/variables/variables0.c:4: ERROR: global variables are not allowed
tests/style/variables/variables0.c:5: ERROR: global variables are not allowed
total: 5 errors, 0 warnings, 12 lines checked
